pom.xml

```diff
@@ -154,16 +155,25 @@
                 <hdp_version>2.6.5.0</hdp_version>
                 <global_hbase_version>1.1.1</global_hbase_version>
                 <global_hbase_guava_version>12.0</global_hbase_guava_version>
+                <!--<global_kafka_version>0.10.0.1</global_kafka_version>-->
+                <!--<global_slf4j_version>1.7.7</global_slf4j_version>-->
+                <global_kafka_version>2.0.0</global_kafka_version>
```

Comment:
Would it make sense to create a separate profile called `HDP-3.1` (or similar) that is active by default that contains the new build versions?

Reply:
Ignore this.  The diff was a little off when I was looking at it.